addons/mail/static/src/components/composer/composer.js

```diff
@@ -48,11 +48,24 @@ class Composer extends Component {
          * Reference of the text input component.
          */
         this._textInputRef = useRef('textInput');
+        /**
+         * Reference of the subject input. Useful to set content.
+         */
+        this._subjectRef = useRef('subject');
         this._onClickCaptureGlobal = this._onClickCaptureGlobal.bind(this);
     }
 
     mounted() {
         document.addEventListener('click', this._onClickCaptureGlobal, true);
+        if(this._subjectRef.el) {
+            this._subjectRef.el.value = this.composer.subjectContent;
+        }
+    }
+
+    patched() {
+        if(this._subjectRef.el) {
+            this._subjectRef.el.value = this.composer.subjectContent;
+        }
```

Comment:
I would define an `_update()` method for shared logic. This is the design approach we decided in some other places, which is good to have component designed in a functional way.